### DISCUSSION TITLE
[Feature] Sorting for search results

### DISCUSSION THREAD

**Discussion by kikendo** [immich-app/immich#12395] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

There's no sorting options whatsoever for search results. I don't understand the order they come out in but it seems random.
Please add the possibility to sort search results, at the very least by date.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8377
